praw/models/listing/generator.py

```diff
@@ -94,6 +94,6 @@ def _next_batch(self):
             raise StopIteration()
 
         if self._listing.after and self._listing.after != self.params.get("after"):
```

Comment:
Does `"after"` here need to change as well?

Reply:
Good catch, fixed